src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c

```diff
@@ -720,7 +735,7 @@ void KaleidoScope_DrawWorldMap(PlayState* play, GraphicsContext* gfxCtx) {
         gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 235, 235, 235, pauseCtx->alpha);
 
         for (k = 0; k < 15; k += 8) {
-            gSPVertex(POLY_OPA_DISP++, &pauseCtx->mapPageVtx[60 + k * 4], 32, 0);
+            gSPVertex(POLY_OPA_DISP++, &pauseCtx->mapPageVtx[(PAGE_BG_QUADS + k) * 4], 32, 0);
```

Comment:
how is adding parens here equivalent?

Reply:
`PAGE_BG_QUADS` expands to `3*5` = 15, and `15*4=60`

Reply:
right but before it's 60 + (k * 4) and now it's (60 + k) * 4

Reply:
(perhaps k * 4 + 60 or 4 * k + 60 would be better if it matches)

Reply:
before:
`60 + k * 4`
after:
`(PAGE_BG_QUADS + k) * 4`
aka `(15 + k) * 4`, is equivalent

Reply:
oh... sorry